src/radical/saga/adaptors/slurm/slurm_job.py

```diff
@@ -502,11 +502,11 @@ def _job_run(self, jd):
         self._logger.debug ("submit SLURM script to %s", self.rm)
         if 'bridges' in self.rm.host.lower():
 
-            if total_gpu_count:
+            if gpu_count:
```

Comment:
How did this work before?  *scratch*

Reply:
I do not know. Also, I found that `total_cpu_count` changed to `cpu_count`